docs/ui/components/Search/styles.ts

```diff
@@ -0,0 +1,862 @@
+import { css } from '@emotion/react';
+import { borderRadius, iconSize, shadows, spacing, theme, typography } from '@expo/styleguide';
+
+import { kbdStyle } from '~/ui/components/Text';
+
+export const DocSearchStyles = css`
+  /* === Variables === */
+
+  :root {
+    --docsearch-primary-color: ${theme.palette.primary[500]};
+    --docsearch-text-color: ${theme.text.default};
+    --docsearch-spacing: ${spacing[4]}px;
+    --docsearch-icon-color: ${theme.icon.secondary};
+    --docsearch-icon-stroke-width: 1.4;
+    --docsearch-highlight-color: var(--docsearch-primary-color);
+    --docsearch-muted-color: rgb(150, 159, 175);
+
+    /* modal */
+    --docsearch-modal-width: 40vw;
+    --docsearch-modal-height: 74vh;
+    --docsearch-modal-background: ${theme.background.default};
+    --docsearch-modal-shadow: ${shadows.popover};
+
+    /* searchbox */
+    --docsearch-searchbox-height: 56px;
+    --docsearch-searchbox-background: ${theme.background.default};
+    --docsearch-searchbox-focus-background: ${theme.background.default};
+    --docsearch-searchbox-shadow: none;
+
+    /* hit */
+    --docsearch-hit-height: 56px;
+    --docsearch-hit-color: ${theme.text.secondary};
+    --docsearch-hit-active-color: ${theme.background.tertiary};
+    --docsearch-hit-background: ${theme.background.default};
+    --docsearch-hit-shadow: none;
+
+    /* footer */
+    --docsearch-footer-height: 44px;
+    --docsearch-footer-background: ${theme.background.secondary};
+  }
+
+  /* Dark Theme overwrites */
+
+  html[data-expo-theme='dark'] {
+    .DocSearch-Hits mark {
+      background: ${theme.palette.primary[200]};
+    }
+
+    .DocSearch-Hit[aria-selected='true'] mark {
+      background: ${theme.palette.primary[300]};
+    }
+
+    .DocSearch-Hit-icon {
+      opacity: 0.6;
+    }
+
+    .DocSearch-Hit-action-button:hover path,
+    .DocSearch-Hit-action-button:focus path {
+      fill: ${theme.palette.primary[600]};
+      stroke: ${theme.palette.primary[600]};
+    }
+
+    .DocSearch-Hit[aria-selected='true'] .DocSearch-Hit-icon:before {
+      filter: brightness(1.25);
+    }
+  }
+
+  /* === Button === */
+
+  .DocSearch-Button {
+    align-items: center;
+    background: var(--docsearch-searchbox-background);
+    border: 0;
+    border-radius: ${borderRadius.small}px;
```

Comment:
```suggestion
    border-radius: ${borderRadius.medium}px;
```
Need a bigger radius here